src/model.ts

```diff
@@ -430,7 +450,11 @@ export class JSORMBase {
   }
 
   get attributes(): Record<string, any> {
-    return this._attributes
+    return Object.assign({}, this.stored, this._attributeOverrides || {})
+  }
+
+  get stored() {
```

Comment:
should this be private?

Reply:
Maybe, I actually like it as a public API though. Could be useful for something like, say, rolling back changes.